crates/ruff_linter/src/rules/pyupgrade/rules/f_strings.rs

```diff
@@ -479,17 +468,27 @@ pub(crate) fn f_strings(
 
     let mut diagnostic = Diagnostic::new(FString, call.range());
 
+    // Avoid refactors that exceed the line length limit or make it exceed by more.
+    let f_string_fits_or_shrinks = fits_or_shrinks(
+        &contents,
+        template.into(),
+        checker.locator(),
+        checker.settings.pycodestyle.max_line_length,
+        checker.settings.tab_size,
+    );
```

Comment:
Honestly I'm tempted to remove this too. I think it will be confusing to users if we don't offer a fix here.

Reply:
agree, but there seems to be an issue with https://github.com/astral-sh/ruff/discussions/7810